src/kirby/components/form-field/form-field.component.scss

```diff
@@ -0,0 +1,39 @@
+@import '../../scss/utils';
+
+:host() {
+  display: block;
+  margin-bottom: size('s');
+}
+
+.texts {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: space-between;
+  padding: 0 size('s');
+
+  .left {
+    flex: 75%;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+  }
+
+  .right {
+    flex: 25%;
+    display: flex;
+    text-align: right;
+    flex-direction: column;
+    justify-content: flex-end;
+  }
+}
+
+label {
+  display: block;
+
+  span {
+    font-size: font-size('s');
+    font-weight: font-weight('light');
+    margin-bottom: size('xxxs');
```

Comment:
@jakobe A floating element can not have margin. So either we have to make this span `display:block` or move the margin to the input elements.

I vote for making the span `display: block`. But i can not remember if we talked about the reason of the span not being `display: block` yestoday?

Reply:
If you need it to be `display: block`, it probably should be a `div` instead as this might cause confusion if you change the basic behavior of `span`.
Btw, how is this a floating element?

Reply:
I meant inline element. Sorry my mistake 😄